<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TransferMoneyPage</class>
 <widget class="QDialog" name="TransferMoneyPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#TransferMoneyPage{
background-color: #eaf2f8;
}</string>
  </property>
  <widget class="QLineEdit" name="secondPassField">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>320</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#secondPassField {
        border: 1px solid #2c3e50;
        border-radius: 6px;
        padding: 6px;
        background-color: #ecf0f1;
        font-size: 14px;
	    color: rgb(0, 0, 0);
 }
 #secondPassField:focus {
        border: 1px solid #2980b9;
        background-color: #ffffff;
    }</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>150</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#label_6 {
        color: #2c3e50;
        font-size: 14px;
        font-weight: bold;
    }</string>
   </property>
   <property name="text">
    <string>به</string>
   </property>
  </widget>
  <widget class="QPushButton" name="returnButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>560</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> #returnButton {
        background-color: #2980b9;
        color: white;
        border: none;
        border-radius: 6px;
        padding: 8px 16px;
        font-size: 14px;
    }
    #returnButton:hover {
        background-color: #3498db;
    }
    #returnButton:pressed {
        background-color: #1c5980;
    }</string>
   </property>
   <property name="text">
    <string>بازگشت</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="expirationDateField">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>460</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#expirationDateField {
        border: 1px solid #2c3e50;
        border-radius: 6px;
        padding: 6px;
        background-color: #ecf0f1;
        font-size: 14px;
	    color: rgb(0, 0, 0);
 }
 #expirationDateField:focus {
        border: 1px solid #2980b9;
        background-color: #ffffff;
    }</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>220</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#label_7 {
        color: #2c3e50;
        font-size: 14px;
        font-weight: bold;
    }</string>
   </property>
   <property name="text">
    <string>مبلغ</string>
   </property>
  </widget>
  <widget class="QLabel" name="toCardNumber">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>150</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#toCardNumber {
        color: #2c3e50;
        font-size: 14px;
        font-weight: bold;
    }</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>460</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#label_11 {
        color: #2c3e50;
        font-size: 14px;
        font-weight: bold;
    }</string>
   </property>
   <property name="text">
    <string>انقضا</string>
   </property>
  </widget>
  <widget class="QLabel" name="amount">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>220</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#amount {
        color: #2c3e50;
        font-size: 14px;
        font-weight: bold;
    }</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="generateDynamicSecondPassButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>320</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> #generateDynamicSecondPassButton {
        background-color: #2980b9;
        color: white;
        border: none;
        border-radius: 6px;
        padding: 8px 16px;
        font-size: 14px;
    }
    #generateDynamicSecondPassButton:hover {
        background-color: #3498db;
    }
    #generateDynamicSecondPassButton:pressed {
        background-color: #1c5980;
    }</string>
   </property>
   <property name="text">
    <string>درخواست رمز دوم پویا</string>
   </property>
  </widget>
  <widget class="QLabel" name="toOwner">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>150</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#toOwner {
        color: #2c3e50;
        font-size: 14px;
        font-weight: bold;
    }</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="fromCardNumber">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>80</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#fromCardNumber {
        color: #2c3e50;
        font-size: 14px;
        font-weight: bold;
    }</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="transferButton">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>560</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> #transferButton {
        background-color: #2980b9;
        color: white;
        border: none;
        border-radius: 6px;
        padding: 8px 16px;
        font-size: 14px;
    }
    #transferButton:hover {
        background-color: #3498db;
    }
    #transferButton:pressed {
        background-color: #1c5980;
    }</string>
   </property>
   <property name="text">
    <string>انتقال</string>
   </property>
  </widget>
  <widget class="QLabel" name="fromOwner">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>80</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#fromOwner {
        color: #2c3e50;
        font-size: 14px;
        font-weight: bold;
    }</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>320</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#label_9 {
        color: #2c3e50;
        font-size: 14px;
        font-weight: bold;
    }</string>
   </property>
   <property name="text">
    <string>رمز دوم</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="cvv2Field">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>390</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#cvv2Field {
        border: 1px solid #2c3e50;
        border-radius: 6px;
        padding: 6px;
        background-color: #ecf0f1;
        font-size: 14px;
	    color: rgb(0, 0, 0);
 }
 #cvv2Field:focus {
        border: 1px solid #2980b9;
        background-color: #ffffff;
    }</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>80</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#label {
        color: #2c3e50;
        font-size: 14px;
        font-weight: bold;
    }</string>
   </property>
   <property name="text">
    <string>از</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>390</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#label_10 {
        color: #2c3e50;
        font-size: 14px;
        font-weight: bold;
    }</string>
   </property>
   <property name="text">
    <string>cvv2</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
